lib/webdriveragent.js

```diff
@@ -108,6 +110,18 @@ class WebDriverAgent {
     });
   }
 
+  async checkForDependencies() {
+    if (await fs.hasAccess(this.bootstrapPath + "/Carthage")) {
+      this.xcodebuild.start();
+      return;
+    }
+    let proc = new SubProcess('/bin/bash', ['Scripts/bootstrap.sh'], {cwd: this.bootstrapPath});
```

Comment:
oh this.xcodebuild is set in line 68 and its just a SubProcess (which wraps teen_process spawn).

```
this.xcodebuild = this.createXcodeBuildSubProcess();
```

start() method is async, but we didnt wait for it previously (line 79)